app/presenters/facet_presenter.rb

```diff
@@ -0,0 +1,11 @@
+class FacetPresenter < SimpleDelegator
+  def initialize(facet, section_index, section_count)
+    super(facet)
+
+    # Facet index is 1-based for GOV.UK analytics
+    @section_index = section_index + 1 if section_index
+    @section_count = section_count
+  end
+
+  attr_reader :section_index, :section_count
```

Comment:
Would suggest moving this to top of the file to be conventional with rest of this apps codebase.